File: Api/Models/Auth/UserResponse.py

from pydantic import BaseModel
from typing import Optional

class UserResponse(BaseModel):
    """
    Pydantic model equivalent to the C# UserResponse DTO.
    Represents the response body for user data.
    """
    id: int
    nome: str
    email: str
    role: str
    avatar: Optional[str] = None
    # Add other properties if they exist in the full C# file

File: Api/Models/Auth/UserResponse.cs
namespace Api.Models.Auth
{
    public class UserResponse
    {
        //Id = user.Id, Nome = user.Nome, Email = user.Email, Role = user.RoleGate

            public int Id {  get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Role {  get; set; }
        public string Avatar { get; set; }
    }
}
